config.toml

```diff
@@ -47,140 +55,323 @@ enableRobotsTXT = true
     name = "Newsletter Archives"
     url = "/newsletter/"
     parent = "Resources"
     weight = 9
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Meeting Minutes"
     url = "/meeting_minutes/"
     parent = "Resources"
     weight = 10
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Blog"
     url = "https://Jakartablogs.ee"
     parent = "Resources"
     weight = 11
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Tech Talk Meet-ups"
     url = "/resources#meetups"
     parent = "Resources"
     weight = 12
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE YouTube Channel"
     url = "https://www.youtube.com/channel/UC4M7h5b6elu9VlzjLfzuXyg"
     parent = "Resources"
     weight = 13
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Specification Projects"
     url = "https://projects.eclipse.org/list-of-projects?combine=jakarta&field_project_techology_types_tid=1102&field_state_value_2=All"
     parent = "Resources"
     weight = 14
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Community Folder"
     url = "https://drive.google.com/open?id=1kJFh6EIpOyAWP3JkymYZXOK311WM2HXm"
     parent = "Resources"
     weight = 15
     
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Jakarta EE Community Calendar"
     url = "/resources#calendar"
     parent = "Resources"
     weight = 16
 
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "Starter for Jakarta EE"
     url = "https://start.jakarta.ee/"
     parent = "Resources"
     weight = 17
     
-[[menu.main]]
+[[languages.en.menu.main]]
     name = "JakartaOne Livestream"
     url = "https://jakartaone.org/"
     parent = "Resources"
     weight = 18
+
+# for Chinese Version below
+
+[languages.zh.Params]
+  google_tag_manager = "GTM-5WLCZXC"
+  description = "Jakarta® Enterprise Edition (EE) 是云原生的未来. Jakarta® EE 开源软件驱动云原生创新, 让企业应用更加现代化，并同时保护Java EE的现有投资."  
+  subtitle = "云原生Java的新家园"
+  seo_title_suffix = " | Eclipse 基金会"
+  keywords = ["Jakarta EE 软件", "Java EE", "cloud", "云", "云原生", "microservices", "微服务", "enterprise", "企业应用", "open source", "开源", "innovation"]
+  logo = "images/jakarta/jakarta-ee-logo.svg"
+  styles = "css/styles.css"
+  favicon = "images/jakarta/favicon.ico"
+  share_img = "images/jakarta/jakarta_ee_400x400.png"
+  gcse = "011805775785170369411:p3ec0igo0qq"
+  subtheme = "jakarta"
+  header_wrapper_class = "header-default-bg-img"
+  js = "js/solstice.js"
+  show_events = true
+  call_for_action_text = "Download"
+  call_for_action_url = "/release/"
+  call_for_action_icon = "fa-download"
+  linkedin_url="https://www.linkedin.com/groups/13597511/"
+  youtube_url="https://www.youtube.com/channel/UC4M7h5b6elu9VlzjLfzuXyg"
+  twitter_url="https://twitter.com/JakartaEE"
+  facebook_url="https://www.facebook.com/JakartaEE"
+  header_left_classes = "col-sm-4"
+  main_menu_wrapper_classes = "col-sm-20 margin-top-10"
+  hide_cfa_same_page = true
+  
+[languages.zh.taxonomies]
```

Comment:
For sections that don't change, we should leave them as is rather than split them like this. It creates work and failure points if we ever need to update these fields